.gitignore

```diff
@@ -1,2 +1,4 @@
-
 node_modules
+./dist
```

Comment:
Although I am happy with it staying in here. I did see another good argument for keeping it in here. If someone wants to just use the package without rebuilding then they can just download it straight from the hub.